test/built-ins/RegExp/prototype/Symbol.match/builtin-coerce-lastindex-err.js

```diff
@@ -0,0 +1,34 @@
+// Copyright (C) 2015 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+description: Behavior when coercion of `lastIndex` attribute throws an error
```

Comment:
We aren't actually testing the coercion here, but the `Get` call for `lastIndex`.